src/app/components/icons/IconSearchBar.tsx

```diff
@@ -0,0 +1,128 @@
+        flexWrap: 'wrap',
+    },
+    searchField: {
+        // flex: '1 1 0px',
+        // minWidth: 300,
+        // marginRight: theme.spacing(2)
+        // borderRadius: theme.shape.borderRadius,
+        // backgroundColor: Colors.white[50],
+        // color: theme.palette.text.primary,
+        // marginTop: theme.spacing(1),
+        // marginRight: theme.spacing(3),
+    },
+    categoryPicker: {
+        // flex: '1 1 0px',
+        // minWidth: 200,
+    },
+}));
+
+type SearchBarProps = HTMLAttributes<HTMLDivElement> & {
+    onSearchChange: (event: ChangeEvent<HTMLInputElement>) => void;
+    onCategoriesChanged: (event: ChangeEvent<{ name?: string; value: any }>) => void;
+    iconCategories: string[];
+};
+export const IconSearchBar: React.FC<SearchBarProps> = (props): JSX.Element => {
+    const { onSearchChange, onCategoriesChanged, iconCategories, ...divProps } = props;
+    const classes = useStyles();
+    const { iconSearch = '' } = useQueryString();
+    const [selectedCategories, setSelectedCategories] = useState<string[]>([]);
+
+    // Media Query adjustments to search spacing
+    const stackSmall = useMediaQuery('(max-width:749px)');
+    const sideBySideSmall = useMediaQuery('(min-width:750px) and (max-width:959px');
+    const stackMedium = useMediaQuery('(min-width:960px) and (max-width:1099px');
+    const sideBySideLarge = useMediaQuery('(min-width:1100px) and (max-width:1259px');
+    // const sideBySideXL = useMediaQuery('(min-width:1260px');
+
+    const searchSize = stackSmall || stackMedium ? 12 : sideBySideSmall || sideBySideLarge ? 6 : 8;
+    const categorySize = stackSmall || stackMedium ? 12 : sideBySideSmall || sideBySideLarge ? 6 : 4;
+
+    const handleFilterChange = useCallback(
+        (e: any): void => {
+            setSelectedCategories(e.target.value);
+            onCategoriesChanged(e);
+        },
+        [setSelectedCategories, onCategoriesChanged]
+    );
+
+    useEffect(() => {
+        onSearchChange({ target: { value: iconSearch } } as ChangeEvent<HTMLInputElement>);
+    }, []);
+
+    return (
+        <Grid container spacing={2} {...divProps}>
+            <Grid item xs={searchSize}>
+                <TextField
+                    className={classes.searchField}
+                    fullWidth
+                    placeholder="Search Icons"
+                    type={'text'}
+                    defaultValue={iconSearch}
+                    onChange={onSearchChange}
+                    variant={'outlined'}
+                    InputProps={{
+                        endAdornment: (
+                            <InputAdornment position={'end'}>
+                                <Search />
+                            </InputAdornment>
+                        ),
+                    }}
+                />
+            </Grid>
+            <Grid item xs={categorySize}>
+                <FormControl className={classes.categoryPicker} variant={'outlined'} fullWidth>
```

Comment:
Should this filter have an icon `startAdornment`?

Reply:
It can. I tweaked the design a little from what was in the original mockup (it felt weird having two different styles of input and the 'group-work' icon didn't quite feel appropriate for categories. Open to suggestions on this (@huayunh @Horchen154 )

Reply:
If you want to do multi-select, use a bunch of inline chips under the search bar. 

![image](https://user-images.githubusercontent.com/8997218/98562904-910e8580-2278-11eb-965f-d1bf0ae2da5d.png)

Reply:
Feels strange to do multi-select in a dropdown.

Reply:
Something like this should have been discussed in the design stage of making this page, not implementation.  I think it's fine for now and if it should be changed that's a separate story.

Reply:
I agree with Evan — if we want to change the design here, we should do it in a separate story. The design that we were working from uses a dropdown.

There are 30 categories right now - that might be a lot of chips to place under the search bar.

Reply:
I would stay away from using chips instead of the dropdown. What would be the use case for selecting multiple options?

Reply:
🤷 greater flexibility than choosing a single category?